docs/proposals/algorithms/lifelong-learning/Unknown Task Recognition Algorithm Reproduction based on Lifelong Learning of Sedna.md

```diff
@@ -0,0 +1,139 @@
+# Unknown Task Recognition Algorithm Reproduction based on Lifelong Learning of Sedna
+Traditional machine learning performs test-set inference by training known samples to which its knowledge is limited. The model which is trained by traditional machine learning cannot be effectively recognized the unknown samples from new classes and they will be processed as known samples. Therefore, the recognition and processing of unknown samples or unknown tasks will become the main research direction of artificial intelligence in the future. This project aims to reproduce the CVPR2021 paper "Learning placeholders for open-set recognition" in the Semantic Segmentation dataset. This paper proposes placeholders that imitate the emergence of new classes, thus helping to transform closed-set training into open-set training.
+
+
+
+## Goals
+1. The reproduction should be completed based on the Cityscapes Semantic Segmentation dataset.
+2. The reproduced code is successfully merged into the lifelong learning module of Sedna.
+3. The recognition accuracy (e.g. f1_score) of unknown classes is greater than 0.9.
```

Comment:
Test report of this algorithm is also needed, e.g., [Testing incremental learning in industrial defect detection](https://ianvs.readthedocs.io/en/latest/proposals/test-reports/testing-incremental-learning-in-industrial-defect-detection-with-pcb-aoi.html).

Reply:
Please follow up this comment.

Reply:
Received. After the embedded work is completed, the test shall be conducted and ended when the test report is obtained.

Reply:
still need follow up